### DISCUSSION TITLE
Custom location using qualifiers (location = /)—please NEVER change that possibility!

### DISCUSSION THREAD

**Discussion by Moonbase59** [NginxProxyManager/nginx-proxy-manager#2786] [General]
I just discovered that one can actually use the same constructs as in Nginx’ configuration files, like entering `= /` in the custom location field, to get _exactly_ only the base URL.

See https://blog.syvi.net/tech-snippets/rewrite-only-the-base-url-to-a-sub-page-using-nginx-proxy-manager.

Please **never** disallow this wonderful possibility for a little more control! Thanks!

**Comment by GerritKuilder** [comment]
Unfortunately the images (and this the most relevant information) are gone.

**Reply by mdrxy** [reply]
Bumping this for @Moonbase59  - same issue here.

**Comment by scyto** [comment]
Interesting, i just moved my handcrafted nginx to npm and couldn't get this to work like it does in nginx (the blog is good, i see the images and text ok).

```
server {
    server_name                 cam.mydomain.com;
    listen                      32802 ssl http2;
    listen                      [::]:32802 ssl http2;
    include                     /etc/nginx/block-exploits.conf;
    access_log                  syslog:server=192.168.1.30,tag=cam,severity=error custom;

# this redirects the base call to cam.mydomain.com > to launch the reequested synology app -  so that you go straight to the app. the use of = / 'literal' ensure all other gets/posts fall to the / catchall lower down.
    location = / {
        return                  301 https://cam.mydomain.com/index.cgi?launchApp=SYNO.SDS.SurveillanceStation;
            }
    location /  {
                proxy_pass              https://172.17.0.1:5101;
                proxy_set_header Host              $server_name; 
                proxy_set_header X-Forwarded-Proto https;
                proxy_set_header Upgrade $http_upgrade;
                proxy_set_header Connection "upgrade";
                proxy_http_version 1.1;
            }
}
```

while this works on normal browsers in both nginx native and npm i see different behavior when connecting via the DSCAM app from synology.  In native nginx this works fine.  In NPM it doesn't, and i can't figure out why.  But it makes me believe there is some issue.
(note in npm i removed the directives not supported in the advanced location field)
